lib/registries/addon/modifiers/with-branding.ts

```diff
@@ -0,0 +1,20 @@
+import Modifier from 'ember-oo-modifiers';
+
+import Brand from 'registries/services/brand';
```

Comment:
oh i missed which `Brand` this was -- i think it'd make more sense to pass a brand model instance rather than the brand service